astropy/modeling/functional_models.py

```diff
@@ -1054,49 +1062,84 @@ class Voigt1D(Fittable1DModel):
     fwhm_L = Parameter(default=2/np.pi)
     fwhm_G = Parameter(default=np.log(2))
 
-    _abcd = np.array([
-        [-1.2150, -1.3509, -1.2150, -1.3509],  # A
-        [1.2359, 0.3786, -1.2359, -0.3786],    # B
-        [-0.3085, 0.5906, -0.3085, 0.5906],    # C
-        [0.0210, -1.1858, -0.0210, 1.1858]])   # D
+    sqrt_pi = np.sqrt(np.pi)
+    sqrt_ln2 = np.sqrt(np.log(2))
+    sqrt_ln2pi = np.sqrt(np.log(2) * np.pi)
+    _last_z = np.zeros(1, dtype=complex)
+    _last_w = np.zeros(1, dtype=float)
+    _wofz = None
+    _last_wofz = None
+
+    def __new__(cls, x_0=x_0.default, amplitude_L=amplitude_L.default, fwhm_L=fwhm_L.default,
+                fwhm_G=fwhm_G.default, method='hum2zpf16c', n_models=None, **kwargs):
+        if str(method).lower() in ('wofz', 'faddeeva', 'scipy'):
+            try:
+                from scipy.special import wofz
+            except (ValueError, ImportError) as err:
+                raise ImportError(f'Voigt1D method {method} requires scipy: {err}.') from err
```

Comment:
I can't believe this is the first time I've seen `from err` used.  Thanks for the education, @dhomeier!  I'll be using this a lot moving forward.

Reply:
Thanks to my Flycheck plugin; I had the look up what the heck it was suggesting.